### ISSUE TITLE
questions about librispeech asr learning rate

### ISSUE THREAD

**Issue by cookingbear** [espnet/espnet#1629]
We have built an e2e training system for librispeech with the same architecture and configuration as in the espnet librispeech recipe. The only difference is that our batch size is smaller(64) and we use the fbank with 40 dims. However, we found that your lr is set to 10.0 which in our system is too large to converge. In our system, lr can only be set to 1.0 but the final acc is lower(about 1.8 points) than that of espnet. So, we wonder that the gap of acc can be caused by learning rate or batch size or fbank dims? And the reason that we can not use 10.0 as our lr is the batch size and fbank configuration?

**Comment by sw005320** [comment]
This is very difficult to answer since the definition of the learning rate is quite different across the implementation. I think it would be better to find the optimal value of your system by yourself. Also, the fbank dimension and batchsize greatly change the performance. But again it depends on which feature extractor you use or how to make a minibatch. I hope you'll find the best configurations by referring our hyperparameters but they are not global parameters.
